src/pages/iou/SplitBillDetailsPage.js

```diff
@@ -0,0 +1,155 @@
+ */
+function getReportID(route) {
+    return route.params.reportID.toString();
+}
+
+/**
+ * Returns all the participants in the active report
+ *
+ * @param {Object} reportAction The IOU split reportAction, which contains the participants
+ * @param {Object} personalDetails The personal details of the users
+ * @return {Array}
+ */
+const getAllParticipants = (reportAction, personalDetails) => {
+    const participants = lodashGet(reportAction, 'originalMessage.participants', []);
+    const participantsExcludingOwner = _.filter(participants, (participant) => participant !== reportAction.actorEmail);
+
+    return _.chain(participantsExcludingOwner)
+        .map((login) => {
+            const userLogin = Str.removeSMSDomain(login);
+            const userPersonalDetail = lodashGet(personalDetails, login, {displayName: userLogin, avatar: ''});
+
+            return {
+                alternateText: userLogin,
+                displayName: userPersonalDetail.displayName,
+                icons: [
+                    {
+                        source: ReportUtils.getAvatar(userPersonalDetail.avatar, login),
+                        name: login,
+                        type: CONST.ICON_TYPE_AVATAR,
+                    },
+                ],
+                keyForList: userLogin,
+                login,
+                text: userPersonalDetail.displayName,
+                tooltipText: userLogin,
+                participantsList: [{login, displayName: userPersonalDetail.displayName}],
+            };
+        })
+        .sortBy((participant) => participant.displayName.toLowerCase())
+        .value();
+};
+
+const SplitBillDetailsPage = (props) => {
+    const reportAction = props.reportActions[`${props.route.params.reportActionID.toString()}`];
+    const participants = getAllParticipants(reportAction, props.personalDetails);
+    const splitAmount = lodashGet(reportAction, 'originalMessage.amount', 0);
+
+    return (
+        <ScreenWrapper includeSafeAreaPaddingBottom={false}>
+            (
+            <FullPageNotFoundView shouldShow={_.isEmpty(props.report) || _.isEmpty(reportAction)}>
+                <HeaderWithCloseButton
+                    title={props.translate('common.details')}
+                    onCloseButtonPress={Navigation.dismissModal}
+                    onBackButtonPress={Navigation.goBack}
+                    shouldShowBackButton={ReportUtils.isChatRoom(props.report) || ReportUtils.isPolicyExpenseChat(props.report) || ReportUtils.isThread(props.report)}
+                />
+                <View
+                    pointerEvents="box-none"
+                    style={[styles.containerWithSpaceBetween]}
+                >
+                    {Boolean(participants.length) && (
+                        <MoneyRequestConfirmationList
+                            hasMultipleParticipants
+                            participants={participants}
+                            iouAmount={splitAmount}
```

Comment:
One bug remains: we'll always show the authenticated user as the payee. I'll fix this tomorrow too